File: curso_c_cpp/c/08_arreglos_y_cadenas/ejemplos/matriz.c
/*
 * Matriz (arreglo 2D) recorrida con dos bucles anidados.
 */
#include <stdio.h>

int main(void) {
    int m[2][3] = {
        {1, 2, 3},
        {4, 5, 6}
    };

    for (int i = 0; i < 2; i++) {          // filas
        for (int j = 0; j < 3; j++) {      // columnas
            printf("%d ", m[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: curso_java/08_arreglos/ejemplos/Matriz.java
/*
 * Matriz (arreglo 2D) recorrida con dos bucles anidados.
 */
public class Matriz {
    public static void main(String[] args) {
        int[][] m = {
            {1, 2, 3},
            {4, 5, 6}
        };

        for (int i = 0; i < m.length; i++) {          // filas
            for (int j = 0; j < m[i].length; j++) {   // columnas de la fila i
                System.out.print(m[i][j] + " ");
            }
            System.out.println();
        }
    }
}
